src/main.rs

```diff
@@ -24,7 +24,6 @@ fn main() -> Result<(), Box<dyn Error>> {
     let conf = config::get_config().unwrap();
     println!("Connecting to ros...");
     rosrust::init("termviz");
-    println!("Retrieving map...");
```

Comment:
unrelated! but good :P 

Should we write for which frames we are waiting? this is causing alot of confusion